.codefresh/codefresh.yaml

```diff
@@ -0,0 +1,44 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
```

Comment:
Can you remove this file? We aren't using codefresh here and it looks like this has some config specific to your fork? 😄

Reply:
Removed. It wasn't specific to my commit. If you are interested though I was going to build a pipeline to run unit tests, start the container to verify that it starts etc and then can do promotion to the registry.